app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
Good point about the `db.create_all()` route not being covered - I forgot that. But yes if we don't use it, then not a problem! 

That cookbook needs more than the skim read I'm giving it, but it seems a little ott. Maybe you can work it out, but I suggest you stick to hacking it all by running sql in the migration scripts - drop the table and create the view. Probably worth a comment explaining it seems the simplest way to create a view with a corresponding ORM model.